src/Core/CMakeLists.txt

```diff
@@ -2,6 +2,11 @@ project(ObEngineCore)
 
 include(group_files)
 
+add_custom_target(ConfigureObEngineGit COMMAND ${CMAKE_COMMAND} 
+    -DGIT_EXECUTABLE=${GIT_EXECUTABLE} -DObEngine_SOURCE_DIR=${ObEngine_SOURCE_DIR} -DObEngine_BINARY_DIR=${ObEngine_BINARY_DIR} -P "${ObEngine_SOURCE_DIR}/cmake/engine_git.cmake")
+file(MAKE_DIRECTORY "${ObEngine_BINARY_DIR}/include/Core/Config/")
+file(TOUCH "${ObEngine_BINARY_DIR}/include/Core/Config/Git.hpp")
```

Comment:
Is it necessary ? I though CMake's `configure_file` automatically generated directory structure / files as needed ?

Reply:
You are right, does not seem to be necessary in this case.